pkg/scheduler/api/devices/nvidia/vgpu/utils.go

```diff
@@ -369,7 +369,7 @@ func checkNodeGPUSharingPredicateAndScore(pod *v1.Pod, gssnap *GPUDevices, repli
 			if int(gs.Device[i].Memory)-int(gs.Device[i].UsedMem) < int(val.Memreq) {
 				continue
 			}
-			if 100-gs.Device[i].UsedCore < uint(val.Coresreq) {
+			if 100-int32(gs.Device[i].UsedCore) < val.Coresreq {
```

Comment:
Any difference with previous code？

Reply:
100-gs.Device[i].UsedCore can be a negative number, you can't assign it to uint type